### PR TITLE
[TS_SELENIUM] Increase logging of tests steps

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Increase logging of tests steps

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/13686

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
crw-ci-test

**Comment by Ohrimenko1988** [comment]
crw-ci-test

File: e2e/tests/e2e_happy_path/HappyPath.spec.ts [hunk 1]
```diff
@@ -125,9 +125,7 @@ suite('Language server validation', async () => {
         try {
             await editor.performKeyCombination(javaFileName, Key.chord(Key.CONTROL, Key.F12));
             await editor.waitEditorAvailable(codeNavigationClassName);
-        }
-        // workaround for issue: https://github.com/eclipse/che/issues/14520
-        catch (err) {
+        } catch (err) {
```
Comment (rhopp) [inline e2e/tests/e2e_happy_path/HappyPath.spec.ts:128]: why was this comment removed?

**Review by rhopp — APPROVED** [review]
lgtm.
just one question: have you tried to look into how to get class/method name automatically? that way we wouldn't need to write the class/method name in the messages by ourselves...

File: e2e/tests/e2e_happy_path/HappyPath.spec.ts (hunk 1 above)
Comment (Ohrimenko1988) [reply e2e/tests/e2e_happy_path/HappyPath.spec.ts:128]: Resolved

**Comment by Ohrimenko1988** [comment]
@rhopp Yes, I tried to find a solution but I didn't find any appropriate solution for automation.

**Comment by rhopp** [comment]
> @rhopp Yes, I tried to find a solution but I didn't find any appropriate solution for automation.

In that case +1 from me ;-)

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by Ohrimenko1988** [comment]
crw-ci-test

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by Ohrimenko1988** [comment]
crw-ci-test

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by Ohrimenko1988** [comment]
crw-ci-test

**Pull request merged**